www/include/common/javascript/datepicker/localizedDatepicker.js

```diff
@@ -134,26 +135,29 @@ function turnOffEvents() {
     $(".timepicker").off('change');
 }
 
+
 function updateEndTime() {
+    var durationScale = { "d" : 86400, "h" : 3600, "m": 60, "s": 1 };
     var start = new Date($(".datepicker").first().val() + ' ' +  $(".timepicker").first().val());
     var end = new Date($(".datepicker").last().val() + ' ' +  $(".timepicker").last().val());
     if (start > end) {
         turnOffEvents();
         var e = new Date();
-        e.setTime(start.getTime() + 7200000); //microseconds
+        e.setTime(start.getTime() + $('#duration').val() * durationScale[$('#duration_scale').val()] * 1000); //milliseconds
```

Comment:
This duration is hardcoded and does not correspond to the one chosen by the user
It may need to be chosen by the user, but the current value does not fit this one

Reply:
I'm not sure to understand.
Here I remove the hardcoded 2 hours duration.
And take by default the one configured in _Administration > Parameters > Monitoring, _Duration_ option.
Of course user can then tune this in the downtime popup window.
Any issue with this ?

Reply:
That's what I'm talking about. The chosen  _Administration > Parameters > Monitoring, Duration_ value isn't this one. I've asked to add another field in the same form, for the user to be able to set this value. But so more work is more critic for now

Reply:
OK, I did not know that _Duration_ value in _Administration > Parameters > Monitoring_ was not intended to be the default downtime duration.
What is it used for then ? Sounds like noting 😕
But I guess you want a per-user tunable.